### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server_Modules_Screen Reader: In Capslock+M mode, Narrator is not reading the role of "Name", "Image/Type" & "status" as "Column Header".

### ISSUE THREAD

**Issue by v-supupp** [Microsoft/IIS.WebManager#81]
**Environment Details:**
URL: https://manage.iis.net/webserver/modules
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
Screen Reader: Narrator, NVDA
**WCAG ID Violated:** WCAG1.3.1

**GitHubTags:** #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ScreenReader; #C+E-H; #.NETWebsites; #Edge; #Chrome; #Narrator; #NVDA; #WCAG1.3.1

**Repro Steps:**
1.	Download and install "Microsoft IIS Administration" application.
2.	Enable Internet information Services.
3.	Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4.	Tab till Web Server and press enter
5.	Tab till Modules and press enter
6.	Start Screen Reader (Narrator/NVDA).
7.	Validate that screen reader is reading the role of column headers.

**Actual Result:**
In Capslock+M mode, Narrator is not reading the role of "Name", "Image/Type" & "status" as "Column Header". Narrator is just reading as “Name Image/Type Status” without mentioning role as “Column Headers”

**Expected Result:**
In Capslock+M mode, Narrator should read the role of "Name", "Image/Type" & "status" as "Column Header.
Narrator should read as “Name Column Header”, “Image/Type column Header” and so on

**Note:**
This issue is observed in all the pages having table headers

**User Impact:**
Users with visual disabilities could not understand them as column headers if screen reader is not reading their role as “column header”

**Attaching video for your reference:**
[Bug_81_Narrator_ColumnHeaders.pptx](https://github.com/Microsoft/IIS.WebManager/files/2318209/Bug_81_Narrator_ColumnHeaders.pptx)

**Comment by jhkimnew** [comment]
duplicate bug to https://github.com/Microsoft/IIS.WebManager/issues/236

**Comment by jhkimnew** [comment]
Fixed

**Comment by fsteffi** [comment]
#A11ySev2;

**Comment by Yashi1201** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
